datameta/resource.py

```diff
@@ -19,8 +19,28 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
+from sqlalchemy import or_
 from uuid import UUID
 
+def resource_query_by_id(db, model, idstring):
```

Comment:
Couldn't we marry the `resource_query_by_id` and `resource_by_id` functions? They seem redundant? Maybe one could use the `resource_query_by_id` inside the `resource_by_id`?

Reply:
Done